pkg/management/management.go

```diff
@@ -15,5 +15,7 @@ type Config struct {
 }
 
 func Start(ctx context.Context, config Config, authZ auth.Config) error {
-	return errors.ThrowUnimplemented(nil, "MANAG-h3k3x", "not implemented yet") //TODO: implement
+	err := api.Start(ctx, config.API)
+	logging.Log("MAIN-6dhRs").OnError(err).Panic("unable to start api")
```

Comment:
panic ...

Reply:
I return the error and remove panic, because in main we have a fatal